docs/source/API/containers/ScatterView.md

```diff
@@ -54,7 +54,7 @@ public:
  ScatterView(std::string const& name, Dims ... dims);
 
  template <typename... P, typename... Dims>
- ScatterView(::Kokkos::Impl::ViewCtorProp<P...> const& arg_prop, Dims... dims);
+ ScatterView(/* is-alloc-prop */ const& arg_prop, Dims... dims);
```

Comment:
Can one readily query the allocation properties (from the constructor) that will be the first argument to ScatterView?  Is it important to be able to query these properties for the purposes of creating the desired ScatterView?

Reply:
They are, for the most part, input-only arguments. We don't store whether the View was initialized, or the execution space used in the construction but we keep the label, of course, and if the View allows padding.